utils/extra_config.py

```diff
@@ -32,3 +54,24 @@ def load_extra_path_config(yaml_path):
                 normalized_path = os.path.normpath(full_path)
                 logging.info("Adding extra search path {} {}".format(x, normalized_path))
                 folder_paths.add_model_folder_path(x, normalized_path, is_default)
+
+def load_server_config(component=None):
```

Comment:
I could not find a global server configuration file, did I miss any ?
Does such a setup is ok for the project ?